examples/apyfixed.py

```diff
@@ -42,6 +42,12 @@
 # Comparisons with integers and floats are supported
 (b >= -2.25, a == 1.75)
 
+# %%
+# Arithmetic with integers and floats is supported by automatically casting the number
+# to the same format of the APyType-object
+b + 3.5
+# Which is equivalent to: b + APyFixed.from_float(3.5, int_bits=3, frac_bits=2)
```

Comment:
It may be that you need a new section here, but not sure.

However, I think doing `int_bits=b.int_bits` etc is a bit clearer.

Reply:
Having "`int_bits=b.int_bits` etc" is clearer, good idea!

I will add another section for now, although it might become a bit verbose.